lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -53,7 +53,7 @@ def build(namespace:, context:, definition:, logger:, statsd_tags:, crd: nil)
 
       def class_for_kind(kind)
         if KubernetesDeploy.const_defined?(kind)
-          KubernetesDeploy.const_get(kind) # rubocop:disable Sorbet/ConstantsFromStrings
```

Comment:
Rubocop locally complains that disabling this cop has no effect, but then Policial fails without it. I added a general exclusion to the main rubocop config instead. We're not using Sorbet.